services/tenant-ui/frontend/src/components/verifier/PresentationDetails.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+    <div v-if="presentation">
+        <ul>
+          <li>Status: {{presentation.status}}</li>
+          <li>Updated at: {{props.presentation.updated_at}}</li>
+          <li>Contact Alias: {{props.presentation.contact.alias}}</li>
+          <hr/>
+            <li v-for="value, attr in props.presentation.acapy.presentation_exchange.presentation?.requested_proof.revealed_attr_groups">
+              <h4>{{attr}}:</h4>
+              <span v-for="val, attr_name in value.values" class="presentation-attr-value">
+                <b>{{attr_name}}</b> : {{val.raw}}
+                <br/>
+              </span>
+          </li>
```

Comment:
No change needed (until we do it more than once), but I think this could be good to do a common component that takes in the object like this and prints it out as we'll probably use that in a lot of 'advanced view' areas